org.fooyin.fooyin.yaml

```diff
@@ -0,0 +1,106 @@
+app-id: org.fooyin.fooyin
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: fooyin
+add-extensions:
+  org.freedesktop.Platform.ffmpeg-full:
+    directory: lib/ffmpeg
+    version: '23.08'
+    add-ld-path: .
+finish-args:
+  - '--device=dri'
+  - '--filesystem=xdg-download'
+  - '--filesystem=xdg-music'
+  - '--filesystem=xdg-run/pipewire-0'
+  - '--filesystem=/run/media'
+  - '--own-name=org.mpris.MediaPlayer2.fooyin'
+  - '--share=ipc'
+  - '--share=network'
+  - '--socket=fallback-x11'
+  - '--socket=pulseaudio'
+  - '--socket=wayland'
+  - '--system-talk-name=org.freedesktop.UDisks2'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--talk-name=org.gtk.Notifications'
+  - '--talk-name=org.kde.StatusNotifierWatcher'
+  - '--talk-name=org.mpris.MediaPlayer2.*'
+cleanup:
+  - '*.a'
+  - '*.la'
+  - '/include'
+  - '/lib/cmake'
+  - '/lib/alsa-lib/pkgconfig'
+  - '/lib/pkgconfig'
+  - '/mkspecs'
+  - '/share/aclocal'
+  - '/share/doc/KDSingleApplication-qt6'
+cleanup-commands:
+  - 'mkdir -p ${FLATPAK_DEST}/lib/ffmpeg'
+modules:
+  - name: alsa-lib
+    buildsystem: simple
+    builddir: true
+    build-commands:
+      - ./configure --prefix=${FLATPAK_DEST} --libdir=${FLATPAK_DEST}/lib/alsa-lib
+      - make -j$FLATPAK_BUILDER_N_JOBS
+      - make install
+    sources:
+      - type: archive
+        url: https://www.alsa-project.org/files/pub/lib/alsa-lib-1.2.11.tar.bz2
+        sha256: 9f3f2f69b995f9ad37359072fbc69a3a88bfba081fc83e9be30e14662795bb4d
+        x-checker-data:
+          type: anitya
+          project-id: 38
+          stable-only: true
+          url-template: https://www.alsa-project.org/files/pub/lib/alsa-lib-$version.tar.bz2
+  - name: KDSingleApplication
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - '-DCMAKE_BUILD_TYPE=Release'
+      - '-DKDSingleApplication_QT6=ON'
+      - '-DKDSingleApplication_EXAMPLES=OFF'
+    sources:
+      - type: git
+        url: https://github.com/KDAB/KDSingleApplication
+        tag: 'v1.1.0'
+        commit: cb0c664b40d3b31bad30aa3521eff603162ed0bd
+        x-checker-data:
+          type: anitya
+          project-id: 370439
+          stable-only: true
+  - name: taglib
```

Comment:
It's in the runtime https://invent.kde.org/packaging/flatpak-kde-runtime/-/blob/4930296f1a6d17eee3863a03c554780bb94baf62/org.kde.Sdk.json.in#L1317

Reply:
Surprising, thanks.